### PR TITLE
Chargeback fix

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0b80957b7a20d9ef8cac7bc93d44b0a69c8457ef:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/intelligence/test_chargeback.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def test_add_new_compute_chargeback():
 def test_compute_chargeback_duplicate_disallowed():
     ccb = new_compute_rate()
     ccb.create()
-    with error.expected('Name already exists'):  # currently cfme just blows up
+    with error.expected('Description has already been taken'):  # currently cfme just blows up
```
Comment (seandst) [inline cfme/tests/intelligence/test_chargeback.py]: That inline comment doesn't make sense to me. Was it saying that CFME currently blows up when we expect 'Name already exists' and now the comment can go away, or is it saying that regardless of the expected error CFME blows up? Or something else entirely?

Comment (psav) [inline cfme/tests/intelligence/test_chargeback.py]: 5.3 still blows up....5.4 did blow up.....it's now been fixed...but the fix didn't align with the code

Comment (seandst) [inline cfme/tests/intelligence/test_chargeback.py]: So does the comment need to be there?

**Pull request merged**